L1Trigger/Phase2L1ParticleFlow/python/L1METPFProducer_cfi.py

```diff
@@ -0,0 +1,6 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
` Note when you use a CFI file in a configuration passed to cmsRun, a manually written CFI file in the python subdirectory will override a generated CFI file with the same file name, so you probably want to remove the existing CFI file from the CMSSW repository when you create a fillDescriptions function that generates a CFI file with the same name. ` 
from https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp#Converting_an_Existing_CFI_file